docs/RC/GetInteriorVehicleDataConsent/index.md

```diff
@@ -0,0 +1,89 @@
+## GetInteriorVehicleDataConsent
+
+Type
+: Function
+
+Sender
+: SDL
+
+Purpose
+: Request for driver's permission to RC module for the specified application
+
+RC.GetInteriorVehicleDataConsent must be sent to HMI in case application in HMILevel FULL requested access to RC module that is already in use by another application. 
+HMI is expected to display a permission prompt to the driver showing the RC module and app details (for example, app's name).
+The driver is expected to have an ability to grant or deny the permission.
+
+!!! must
```

Comment:
I don't think "!!! must" is the right tag for these bullet points. These sentences are not descriptive for what the HMI must do...

Reply:
@JackLivio do you mean steps should describe HMI behavior?

Reply:
Yes the must should describe the HMI's absolute responsibilities when handling this rpc. for example, HMI `must` prompt user to make selection for corresponding App and module after receiving GetInteriorVehicleDataConsent.

Reply:
@LuxoftAKutsan ^